authors/apps/authentication/backends.py

```diff
@@ -1,10 +1,69 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+import jwt
 
-"""Configure JWT Here"""
+from django.conf import settings
 
+from rest_framework import authentication, exceptions
+
+from .models import User
+
+
+class JWTAuthentication(authentication.BaseAuthentication):
+    authentication_header_prefix = 'Token'
+
+    def authenticate(self, request):
```

Comment:
Why do you have this method? I think you can achieve whatever your doing here in `authenticate_user_details`